server/memphis_handlers_stations.go

```diff
@@ -108,6 +108,8 @@ func removeStationResources(s *Server, station models.Station) error {
 		return err
 	}
 
+	DeleteTagsByStation(station.ID)
```

Comment:
I can't comment in the right place but under the DeleteTagsByStation implementation there is the same function in comment remove it

Reply:
it's delete tags by schema under it